src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/HaFlowMapper.java

```diff
@@ -101,11 +101,17 @@ public abstract class HaFlowMapper {
     @Mapping(target = "flowId", ignore = true)
     @Mapping(target = "timeCreate", ignore = true)
     @Mapping(target = "timeUpdate", ignore = true)
+    @Mapping(target = "forwardLatency", ignore = true)
+    @Mapping(target = "reverseLatency", ignore = true)
+    @Mapping(target = "latencyLastModifiedTime", ignore = true)
     public abstract HaSubFlowDto toSubFlowDto(HaSubFlowCreatePayload source);
```

Comment:
It looks like we do not use this method,
may we remove it?

Reply:
We use it at org/openkilda/northbound/converter/HaFlowMapperImpl.java:535